### PR TITLE
fix: dupe payment request height

### LINKED ISSUE #1923: Hide the "Size of the button displayed for Express Checkouts" button
**Describe the bug**
The "Size of the button displayed for Express Checkouts" button was introduced for the new settings UI. In the old UI, we continue to let users choose whichever height they want to set. We want to hide the new express checkout button, but keep the old "payment request button":
image

**To Reproduce**
Steps to reproduce the behaviour:
1. This applies to both situations where `_wcstripe_feature_upe` feature flag is enabled or disabled.
2. Go to `wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`
3. Scroll down to "Payment Request Button Height" 
4. See both "Payment Request Button Height" and "Size of the button displayed for Express Checkouts" 

**Expected behaviour**
Only shows "Payment Request Button Height"

**Screenshots**

### PR DESCRIPTION
**Pull request by frosso**
# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1923

Removes the duplicated height based on the UPE feature flag.

# Testing instructions
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- The height/size attribute on the payment request section should no longer appear duplicated

### CODE REVIEW

File: assets/js/stripe-admin.js [hunk 1]
```diff
@@ -54,9 +54,9 @@ jQuery( function( $ ) {
 			// Toggle Payment Request buttons settings.
 			$( '#woocommerce_stripe_payment_request' ).on( 'change', function() {
 				if ( $( this ).is( ':checked' ) ) {
-					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
+					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_size #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
```
Comment (frosso) [inline assets/js/stripe-admin.js]: This also ensures that we hide it when the "Payment request" checkbox is enabled/disabled

Before:


After:

File: includes/admin/stripe-settings.php [hunk 2]
```diff
@@ -197,7 +197,7 @@
 			],
 		],
 		'payment_request_button_size'         => [
-			'title'       => __( 'Size of the button displayed for Express Checkouts', 'woocommerce-gateway-stripe' ),
+			'title'       => __( 'Payment Request Button Size', 'woocommerce-gateway-stripe' ),
```
Comment (frosso) [inline includes/admin/stripe-settings.php:200]: Adjusted the naming for consistency

File: assets/js/stripe-admin.js [hunk 3]
```diff
@@ -54,9 +54,9 @@ jQuery( function( $ ) {
 			// Toggle Payment Request buttons settings.
 			$( '#woocommerce_stripe_payment_request' ).on( 'change', function() {
 				if ( $( this ).is( ':checked' ) ) {
-					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
+					$( '#woocommerce_stripe_payment_request_button_theme, #woocommerce_stripe_payment_request_button_type, #woocommerce_stripe_payment_request_button_locations, #woocommerce_stripe_payment_request_button_size, #woocommerce_stripe_payment_request_button_height' ).closest( 'tr' ).show();
```
Comment (frosso) [inline assets/js/stripe-admin.js:57]: This also ensures that we hide it when the "Payment request" checkbox is enabled/disabled

Before:


After:

**Review by harriswong — APPROVED** [review]
LGTM 👍 

- Tested when `_wcstripe_feature_upe` is `false`, only "Payment Request Button Height" shows.
- Tested when `_wcstripe_feature_upe` is `true`, only "Payment Request Button Size" shows.

**Pull request merged**